libp2p/Host.cpp

```diff
@@ -526,25 +520,29 @@ void Host::addNode(NodeID const& _node, NodeIPEndpoint const& _endpoint)
         return;
     }
 
-    if (_endpoint.tcpPort() < 30300 || _endpoint.tcpPort() > 30305)
+    if (_endpoint.tcpPort() < c_minListenPort || _endpoint.tcpPort() > c_maxListenPort)
```

Comment:
Why this check is important?

Reply:
I guess it's not, we could remove this if and log

Reply:
I was wondering what the importance of that check was, will remove it